html/forms/tasks/advanced-styling/marking.md

```diff
@@ -48,7 +48,7 @@ input[type="search"] {
   -webkit-appearance: none;
   border-radius: 6px;
 
-  outline: none;
+  outline: transparent;
```

Comment:
The default value of `outline-style` is `none`. So, using the shorthand, setting `outline: transparent`, sets the outline-style to `none`. The original was better.